libmamba/src/core/solver.cpp

```diff
@@ -28,11 +28,11 @@
 namespace mamba
 {
     MSolver::MSolver(MPool pool, const std::vector<std::pair<int, int>> flags)
-        : m_flags(std::move(flags))
-        , m_is_solved(false)
+        : m_libsolv_flags(std::move(flags))
```

Comment:
This results in a copy because `flags` is `const` and `std::vector` does not provide a constructor accepting a const rvalue reference.